codex/conf.nim

```diff
@@ -126,6 +127,13 @@ type
       abbr: "d"
       name: "data-dir" }: OutDir
 
+    circuitDir* {.
```

Comment:
There are specific parametters to indicate where each circuit file is located, we should repoint the defaults to this directory and/or remove the individual circuit paths.

Reply:
I have also moved this option into the `prover` switch. 👍